cfme/infrastructure/host.py

```diff
@@ -402,16 +402,28 @@ def equal_drift_results(self, row_text, section, *indexes):
         tb.select("Select up to 10 timestamps for Drift Analysis")
 
         # Make sure the section we need is active/open
-        sec_loc_map = {
-            'Properties': 'Properties',
-            'Security': 'Security',
-            'Configuration': 'Configuration',
-            'My Company Tags': 'Categories'}
-        active_sec_loc = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]"\
-            "/span[contains(@class, 'dynatree-selected')]".format(sec_loc_map[section])
-        sec_checkbox_loc = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]"\
-            "//span[contains(@class, 'dynatree-checkbox')]".format(sec_loc_map[section])
-        sec_apply_btn = "//div[@id='accordion']/a[contains(normalize-space(text()), 'Apply')]"
+        if version.current_version() >= '5.7':
+            sec_loc_map = {
+                'Properties': 'Properties',
+                'Security': 'Security',
+                'Configuration': 'Configuration',
+                'My Company Tags': 'Categories'}
+            active_sec_loc = "//div[@id='all_sectionsbox']//li[@title='Categories']"\
```

Comment:
**Required** 
This is now hard coded to locate the 'Categories' list item, which is 'My Company Tags'. There is no formatting metachar, `{}`, in the xpath string,  so the `format()` does nothing here. It should format the sec_loc_map value in where 'Categories' is in the xpath.

Won't break for the test you're changing now, but it'll break for any other section.